<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Генератор расписания</string>
  </property>
  <widget class="QDateEdit" name="dateYearStart">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>110</width>
     <height>0</height>
    </size>
   </property>
   <property name="date">
    <date>
     <year>2014</year>
     <month>9</month>
     <day>1</day>
    </date>
   </property>
   <property name="displayFormat">
    <string>dd.MM.yyyy</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateYearFinish">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
   <property name="date">
    <date>
     <year>2015</year>
     <month>5</month>
     <day>31</day>
    </date>
   </property>
   <property name="displayFormat">
    <string>dd.MM.yyyy</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QListWidget" name="Excludes">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>90</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAddExclude">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateExcludeStart">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>60</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
   <property name="date">
    <date>
     <year>2014</year>
     <month>9</month>
     <day>1</day>
    </date>
   </property>
   <property name="displayFormat">
    <string>dd.MM.yyyy</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateExcludeFinish">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>60</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
   <property name="date">
    <date>
     <year>2014</year>
     <month>9</month>
     <day>1</day>
    </date>
   </property>
   <property name="displayFormat">
    <string>dd.MM.yyyy</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClearExcludes">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>30</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnClearExcludes</sender>
   <signal>clicked()</signal>
   <receiver>Excludes</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
